contracts/argent_account.cairo

```diff
@@ -118,4 +145,69 @@ mod ArgentAccount {
         }
         ecdsa::check_ecdsa_signature(hash, guardian_backup::read(), signature_r, signature_s)
     }
+
+    #[external]
+    fn trigger_escape_signer() {
+        // only called via execute
+        asserts::assert_only_self();
+
+        // no escape when the guardian is not set
+        let guardian = guardian::read();
+        assert(guardian != 0, 'argent: guardian required');
+
+        // no escape if there is a guardian escape triggered by the signer in progress
+        let current_timestamp = escape_active_at::read();
+        let current_type = escape_type::read();
+        assert(current_timestamp == 0 & current_type == 0, 'argent: cannot override escape');
```

Comment:
i think this is not correct, should be comparing to a signer escape type (and with an if statement I find it easier to audit/understand

```suggestion
        if current_timestamp != 0 {
                assert(current_type == ESCAPE_TYPE_SIGNER, 'argent: cannot override escape');
        }
```